src/core/reionization.c

```diff
@@ -1733,6 +1735,90 @@ void save_reion_input_grids(int snapshot)
   mlog("...done", MLOG_CLOSE);
 }
 
+
+void load_reion_sfr_grids(int snapshot, float weight, const int new_load)
+{
```

Comment:
I have a doubt in this function, are we actually modifying the sfr grids that we created in reionization.c? Does this mean that the saved ones will also change?

Reply:
i dont think so, we are only reading the hdf5 file with the flag: H5F_ACC_RDONLY